addons/l10n_my_ubl_pint/models/res_company.py

```diff
@@ -0,0 +1,17 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import models, fields
+
+
+class ResCompany(models.Model):
+    _inherit = 'res.company'
+
+    sst_registration_number = fields.Char(related='partner_id.sst_registration_number', readonly=False)
+    ttx_registration_number = fields.Char(related='partner_id.ttx_registration_number', readonly=False)
+
+
+class BaseDocumentLayout(models.TransientModel):
+    _inherit = 'base.document.layout'
+
+    account_fiscal_country_id = fields.Many2one(related="company_id.account_fiscal_country_id")
+    sst_registration_number = fields.Char(related='company_id.sst_registration_number')
+    ttx_registration_number = fields.Char(related='company_id.ttx_registration_number')
```

Comment:
Sorry, I'm unsure at what you are asking ^^
Both of these are used in the invoice report on the company side, and SST is used on the customer side.

SST is also used for pint, but TTx not yet because we don't have TTx taxes/nobody ever asked for them.
Still, it's better to have the TTx number here now in case we need it later, a lot easier to adapt then.

Reply:
My bad, I thought `company` was a res.company object, instead of a `base.document.layout`

Reply:
Ah yes, the template is doing weird thing with the name at some places ^^